src/domain/entities/Program.ts

```diff
@@ -65,6 +82,41 @@ export interface ProgramMetadata {
     options: Option[];
     trackedEntityAttributes?: TrackedEntityAttibute[];
     programStages: ProgramStage[];
+    programRules: D2ProgramRule[];
+    programRuleVariables: D2ProgramRuleVariable[];
+    programRuleActions: D2ProgramRuleAction[];
+}
+
+export interface D2ProgramRule {
```

Comment:
D2 entities better in the D2 repository because DHIS2 concepts should be avoided outside the repos and should not be known from the domain.

Reply:
done